### PR TITLE
Create Moyo Header

### PR DESCRIPTION
**Pull request by SerhiiKirik**
- [DEMO LINK](https://SerhiiKirik.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://SerhiiKirik.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,56 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+header {
```
Comment (illiarerun) [inline src/style.css:5]: Create class for header, don't put style on tags

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,69 @@
...
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="" class="hrefLogo">
```
Comment (illiarerun) [inline src/index.html:24]: You should stick with snake_case
```suggestion
      <a href="" class="header__logo">
```

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,69 @@
...
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="moyoLogo"
+        >
+      </a>
+      <nav>
+        <ul class="nav_navigation">
```
Comment (illiarerun) [inline src/index.html:32]: nav__menu would be better name

File: src/index.html [hunk 4]
```diff
@@ -6,9 +6,69 @@
...
+          <li class="nav__item">
+            <a href="#apple" class="nav__list is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__list">Samsung</a>
```
Comment (illiarerun) [inline src/index.html:38]:
```suggestion
            <a href="#samsung" class="nav__list">Samsung</a>
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,56 @@
...
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav__item {
+  font-family: "Roboto", sans-serif;
```
Comment (illiarerun) [inline src/style.css:15]: Move it to html tag, font should be the same for the whole page

File: src/style.css [hunk 6]
```diff
@@ -1 +1,56 @@
...
+  list-style: none;
+  font-size: 12px;
```
Comment (illiarerun) [inline src/style.css:17]: This too

File: src/style.css [hunk 7]
```diff
@@ -1 +1,56 @@
...
+  display: inline-block;
```
Comment (illiarerun) [inline src/style.css:18]: Why inline-block? It's unnecessary, you have flex wrapper
```suggestion
  display: block;
```

File: src/style.css [hunk 8]
```diff
@@ -1 +1,56 @@
...
+
+.nav__list {
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  display: block;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 16px;
```
Comment (illiarerun) [inline src/style.css:32]: Now, as you removed inline-block, it should be 20

File: src/style.css [hunk 9]
```diff
@@ -1 +1,63 @@
...
+
+.nav__item {
+  list-style: none;
+  display: block;
+  text-transform: uppercase;
+  font-weight: 500;
+  position: relative;
+}
+
+.nav__list {
```
Comment (illiarerun) [inline src/style.css:31]:
```suggestion
.nav__link {
```

**Comment by manch0ffline** [comment]
[DEMO LINK](https://manch0ffline.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://manch0ffline.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**